lms/templates/static_templates/contact.html

```diff
@@ -0,0 +1,37 @@
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Contact page")}</%block>
+
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="container about">
+        <h1>${_("Contact page")}</h1>
+        <p>
+            ${_("Welcome to the ACCEL platform, if you have any questions, wishes or problems, we will be happy to help.")}
+        </p>
+        % if sent_mail:
+        <div class="group group-form group-form-requiredinformation">
+            <h4>${_("Your message was successfully sent.")}</h4>
+        </div>
+        % endif
+        <div class="group group-form group-form-requiredinformation">
```

Comment:
Do we still need an outer object if the form is absent?

Reply:
I removed this div